<ui version="4.0" >
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find and Replace</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>Find &amp;text:</string>
     </property>
     <property name="buddy" >
      <cstring>findText</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="findText" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="insertPolicy" >
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="autoCompletion" >
      <bool>false</bool>
     </property>
     <property name="duplicatesEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="0" column="2" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="findNextButton" >
       <property name="text" >
        <string>Find &amp;Next</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findPrevButton" >
       <property name="text" >
        <string>Find &amp;Prev</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton" >
       <property name="text" >
        <string>&amp;Replace</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton" >
       <property name="text" >
        <string>Replace &amp;All</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="replaceTextLabel" >
     <property name="text" >
      <string>Replace te&amp;xt:</string>
     </property>
     <property name="buddy" >
      <cstring>replaceText</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="replaceText" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="insertPolicy" >
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="autoCompletion" >
      <bool>false</bool>
     </property>
     <property name="duplicatesEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QGroupBox" name="optionsBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="caseCheckBox" >
        <property name="text" >
         <string>&amp;Match upper/lower case</string>
        </property>
        <property name="shortcut" >
         <string>Alt+M</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="wordCheckBox" >
        <property name="text" >
         <string>Whole &amp;word</string>
        </property>
        <property name="shortcut" >
         <string>Alt+W</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="regexpCheckBox" >
        <property name="text" >
         <string>Regular &amp;Expression</string>
        </property>
        <property name="shortcut" >
         <string>Alt+E</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="wrapCheckBox" >
        <property name="text" >
         <string>Wrap aroun&amp;d</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>findText</tabstop>
  <tabstop>replaceText</tabstop>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>wordCheckBox</tabstop>
  <tabstop>regexpCheckBox</tabstop>
  <tabstop>wrapCheckBox</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>findPrevButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>503</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel" >
     <x>544</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
